lib/Site.js

```diff
@@ -253,9 +268,12 @@ Site.prototype.collectUserDefinedVariablesMap = function () {
   // The key is the base directory of the site/subsites,
   // while the value is a mapping of user defined variables
   this.userDefinedVariablesMap = {};
+  const glyphiconsMap = getGlyphiconsMap();
 
   Object.keys(this.baseUrlMap).forEach((base) => {
     const userDefinedVariables = {};
+    Object.assign(userDefinedVariables, glyphiconsMap);
```

Comment:
Putting it here seems to be mixing the logic between user defined variables and the glyph icons. Should we be creating a separate method specially for our custom built-in syntax?

Reply:
I considered that too. Perhaps `this.userDefinedVariablesMap` could be renamed `this.contextVariablesMap`. I couldn't come up with a good name for the separate method though. I wanted to avoid the `collect` prefix as it's distinctly different from the `collect*` methods that are user-dependent, but I'm open to suggestions.

Reply:
Perhaps `setupDefaultContextVariablesMap()`. Then default variables such as this one (the glyph icons) and others (e.g. in the future, [`{{timestamp}}`](https://github.com/MarkBind/markbind/issues/187)) can be set up in this method instead.